### DISCUSSION TITLE
In my CMD it gives a warning like this. How do I fix it?

### DISCUSSION THREAD

**Discussion by likemork** [richrobber2/canvas-zoom#97] [Q&A] [answered]
![Screenshot 2024-01-05 095043](https://github.com/richrobber2/canvas-zoom/assets/39330440/de1a366c-8e6b-4dc5-9be3-251a18f56cdf)
I use https://github.com/EmpireMediaScience/A1111-Web-UI-Installer where it automatically updates extensions Then there will be a notification as shown in this picture. which I don't know how to fix.

**Comment by daswer123** [comment, marked as answer]
Hi, delete this file and try to run it again.

**Reply by likemork** [reply]
It works.Thanks
